source/github-helpers/get-default-branch.ts

```diff
@@ -24,10 +24,14 @@ const getDefaultBranch = cache.function(async function (repository?: pageDetect.
 
 	if (arguments.length === 0 || JSON.stringify(repository) === JSON.stringify(getRepo())) {
 		if (pageDetect.isRepoHome()) {
-			const currentBranch = await elementReady('[data-hotkey="w"]');
-			if (currentBranch) { // If missing, it'll default to the API call
-				return currentBranch.title;
+			const branchSelector = await elementReady('[data-hotkey="w"]');
+			if (branchSelector) {
+				return branchSelector.title === 'Switch branches or tags'
+					? branchSelector.textContent!.trim()
+					: branchSelector.title;
 			}
+
+			console.error(`Could not find the branch selector. Calling API to get the default branch of ${repository.nameWithOwner}.`);
```

Comment:
Why do we need the user to see this? If you need it for debugging we have a logger.

Reply:
Maybe I misunderstood what @fregante means 🤷 the reason I do this is more about unwilling to import `features` here.

But perhaps I should just export and import `logError` here...